client/src/components/datasetCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import * as styles from './datasetCard.module.scss';
+
+interface IDatasetCardProps {
+  key: number,
+  datasetCardProps: { [key:string]: string }
+}
+
+const DatasetCard = ({key, datasetCardProps}:IDatasetCardProps) => {
+  console.log(key);
+  return (
+    <div className={styles.datasetCard}>
+      <h3 className={styles.datasetCardIndicator}>{datasetCardProps.indicator}</h3>
+      <div className={styles.datasetCardWhatIsIt}>What is it?</div>
+      <div className={styles.datasetCardDescription}>
+        {datasetCardProps.description}
+      </div>
+
+      <ul className={styles.datasetCardList}>
+        <li className={styles.datasetCardListItem}>
+          <span className={styles.datasetCardLabels}>
+            Data resolution:&nbsp;
```

Comment:
Two questions about this one:
1. Can we move the `nbsp` to css ?
2. Can we use `formatMessage`?

Reply:
great call on the `nbsp`! and yes to `formatMessage`

Reply:
nbsp = not being super productive (lazy) 🤣